scripts/india_rbi/below_poverty_line/README.md

```diff
@@ -0,0 +1,77 @@
+# Number and Percentage of Population Below Poverty Line
+
+
+## About the Dataset
+Number and Percentage of Population Below Poverty Line published by Reserve Bank of India/Planning Commission.
+
+
+### Download URL
+
+* Available for downlod as [XLSX](https://rbidocs.rbi.org.in/rdocs/Publications/DOCs/154T_HB15092019609736EE47614B23BFD377A47FFC1A5D.XLSX)
+
+
+### Overview
+Povery data is per state and by year. There are three sets of data, for the years 2004-05, 2009-10, 2011-12. The dataset contains the following columns.
+
+1. Year  
+2. State/Union Territory
+3. No. of Persons (Thousands), Rural  
+4. % of Persons, Rural
+5. Poverty line (₹), Rural  
+6. No. of Persons (Thousands), Urban  
+7. % of Persons, Urban
+8. Poverty line (₹), Urban  
+9. No. of Persons (Thousands), Combined
+10. % of Persons, Combined
+
+
+**Notes from the RBI/Planning Commission data page :** 
+1. Population as on 1st March 2012 has been used for estimating number of persons below poverty line. (2011 Census population extrapolated)
+2. Poverty line of Tamil Nadu has been used for Andaman and Nicobar Island.
+3. Urban Poverty Line of Punjab has been used for both rural and urban areas of Chandigarh.
```

Comment:
Since there IS a concept of "Urban Poverty Line" of XXXPlace

We could make urbanPovertyLine, ruralPovertyLine, and povertyLine direct properties of the places. Then we would not need the StatVars `PovertyLine_Person_Urban` `PovertyLine_Person_Rural`

Reply:
Done. As per our discussion.